src/testing/code-debugging.md

```diff
@@ -644,9 +656,11 @@ objects to the method's argument, and call the superclass method.
 
 If you are trying to debug a compositing issue, you can use
 [`debugDumpLayerTree()`][].
-For the previous example, it would output:
 
-```
+<details markdown="1">
+<summary>Expand the layer tree for Example 1</summary>
```

Comment:
Same as above, perhaps "Expand to view the" or "Expand for" would be more accurate? Since it's not the layer tree itself being expanded.

```suggestion
<summary>Expand to view the output layer tree for Example 1</summary>
```